pkg/config/example_test.go

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config_test
+
+import (
+	"os"
+
+	"github.com/golang/protobuf/proto"
+	cb "github.com/hyperledger/fabric-protos-go/common"
+	"github.com/hyperledger/fabric/common/tools/protolator"
+	"github.com/hyperledger/fabric/pkg/config"
+)
+
+func ExampleNewCreateChannelTx() {
+	channel := &config.Channel{
+		ChannelID:  "testChannel",
+		Consortium: "SampleConsortium",
+		Application: &config.Application{
+			Organizations: []*config.Organization{
+				{
+					Name: "Org1",
+				},
+				{
+					Name: "Org2",
+				},
+			},
+			Capabilities: map[string]bool{"V1_3": true},
+			ACLs:         map[string]string{"event/Block": "/Channel/Application/Readers"},
+			Policies: map[string]*config.Policy{
+				config.ReadersPolicyKey: {
+					Type: config.ImplicitMetaPolicyType,
+					Rule: "ANY Readers",
+				},
+				config.WritersPolicyKey: {
+					Type: config.ImplicitMetaPolicyType,
+					Rule: "ANY Writers",
+				},
+				config.AdminsPolicyKey: {
+					Type: config.ImplicitMetaPolicyType,
+					Rule: "MAJORITY Admins",
+				},
+				config.EndorsementPolicyKey: {
+					Type: config.ImplicitMetaPolicyType,
+					Rule: "MAJORITY Endorsement",
+				},
+				config.LifecycleEndorsementPolicyKey: {
+					Type: config.ImplicitMetaPolicyType,
+					Rule: "MAJORITY Endorsement",
+				},
+			},
+			Resources: &config.Resources{DefaultModPolicy: config.AdminsPolicyKey},
```

Comment:
I'm not sure what resources is actually for. Opened https://jira.hyperledger.org/browse/FAB-17596 to investigate its actual usage

Reply:
I talked to Jason and it sounds like we can remove this since Resources isn't actually used as a value. Feel free to remove it from the example and I'll remove it from the config package entirely as part of [FAB-17596](https://jira.hyperledger.org/browse/FAB-17596)

Reply:
done